image/templates/helm/stackrox-central/templates/01-central-12-central-db.yaml

```diff
@@ -134,7 +134,11 @@ spec:
         {{- toYaml ._rox.central.db.persistence._volumeCfg | nindent 8 }}
       - name: config-volume
         configMap:
+          {{- if ._rox.central.db.configOverride }}
+          name: {{ ._rox.central.db.configOverride }}
```

Comment:
Nit: there's probably some template function which acts like `foo if foo ! "" else bar` that could help avoid the if/else here?

Reply:
Trying a sprig ternary, let's see. I do find the if/else very annoying as well